setup/so-functions

```diff
@@ -3057,6 +3057,20 @@ zeek_logs_enabled() {
                         "    - opcua_binary_browse_response_references"\
                         "    - opcua_binary_browse_diagnostic_info"\
                         "    - opcua_binary_create_subscription"\
-                        "    - opcua_binary_read" >> "$zeeklogs_pillar"
+                        "    - opcua_binary_read"
```

Comment:
Does this line need a trailing backslash?

Reply:
It does.  Correction complete.  Thank you.